packages/paste-website/src/pages/libraries/codemods/index.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Codemods
+description: A collection of jscodeshift codemods to help update projects built with Paste.
+slug: /libraries/codemods/
+---
+
+import {graphql} from 'gatsby';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+
+export const pageQuery = graphql`
+  {
+    allPasteLibraries(filter: {name: {eq: "@twilio-paste/codemods"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/libraries/codemods/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Codemods"
+  categoryRoute={SidebarCategoryRoutes.LIBRARIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-codemods"
+  data={props.data.allPasteLibraries.edges}
+/>
+
+---
+
+<contentwrapper>
+<PageAside data={props.data.mdx} />
+<content>
+
+## About
+
+This library provides codemods to alleviate the burden of teams making large sweeping changes related to Paste.
+When dealing with changes that affect many files in a project, we typically we have to
+make a trade-off between the benefit of the change and the risk of introducing errors and tedium.
```

Comment:
> we typically we

Small typo.

I also feel like this sentence is missing the punchline or leaves me hanging. What ties _our_ changes to many files, the trade-offs, introducing errors and tedium to codemods?